node-notes-cli/app.js

```diff
@@ -0,0 +1,60 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+// define operations
+const getOperation = { read, create, delete: deleteEntry, update };
+// try catch phases
+
+try {
+  let data = await readFile('data.json', 'utf-8');
+  data = JSON.parse(data);
+  const operation = process.argv[2];
+  if (Object.keys(getOperation).includes(operation)) {
+    getOperation[operation](data);
+  } else {
+    throw new Error(
+      `Operation ${operation} is not defined. Try create, read, update, delete only`
+    );
```

Comment:
You could say `Try ${getOperation.keys()} only`. Or `Try ${getOperation.keys().join()} only`